riscv-asm.md

```diff
@@ -525,6 +528,82 @@ Integers in the subinterval [-2048, -1] can also be passed by their (unsigned) a
 in the interval [0xfffff800, 0xffffffff] on RV32I, and
 in [0xfffffffffffff800, 0xffffffffffffffff] on both RV32I and RV64I.
 
+Load Floating Point Immediate
+-------------------
+
+`Zfa` extension has provide `fli.{h|s|d|q}` instruction for loading a specific
+set of floating-point immediates.
+
+The supported floating point immediates are listed in the following table:
+
+Value                       | Hexadecimal                    | Decimal               | Standard Representation / Scientific Notation
+:----------------------     | :---------------               | :-------------------  |
+-1.0                        |-0x1p+0                         | -1.0                  | -1.000000e+00
+Minimum positive normal     | min                            | min                   | min
+1.0 x 2 ^ -16               | 0x1p-16                        | 0.0000152587890625    | 1.525879e-05
```

Comment:
Re: `1.525879e-05`: I thought we had agreed to disallow inexact constants, so that this variant would be rejected but `1.52587890625e-06` would be accepted?

Reply:
My bad, this table is generated from my C code, I did check output for decimal but not well check on Scientific Notation form

Reply:
Cool.